modules/gltf/gltf_document.cpp

```diff
@@ -4490,6 +4500,16 @@ Error GLTFDocument::_parse_texture_samplers(Ref<GLTFState> p_state) {
 	return OK;
 }
 
+inline void _set_material_texture_name(const Ref<Texture2D> &p_texture, const String &mat_name, const String &path, const String &p_suffix) {
```

Comment:
Considering how `mat_name` and `p_suffix` are used together, these should be grouped. Also, parameter names must start with `p_`.

```suggestion
inline void _set_material_texture_name(const Ref<Texture2D> &p_texture, const String &p_path, const String &p_mat_name, const String &p_suffix) {
```